pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "json-to-avro"
+version = "0.1.0a1"
+description = "Convert arbitrary JSON data to avro serialized data, registering transitively backwards compatible schemas with Kafka Schema Registry along the way."
+authors = ["Brad Boggs <bboggs@streambit.software>"]
+license = "MIT"
```

Comment:
I went with MIT because the python-schema-registry-client uses it. I figure it's simpler to keep them aligned.